contentcuration/contentcuration/management/commands/exportchannel.py

```diff
@@ -191,49 +193,55 @@ def create_perseus_exercise(ccnode):
         )
         logging.debug("Created exercise for {0} with checksum {1}".format(ccnode.title, assessment_file_obj.checksum))
 
+placeholder="${u'\u0420' IMAGEREPLACE}/".encode('utf-8')
```

Comment:
Should this be in our shared utility repo? Also, can you add a comment showing what this uniform character is?